localstack/services/apigateway/apigateway_listener.py

```diff
@@ -405,8 +406,12 @@ def invoke_rest_api_integration_backend(api_id, stage, integration, method, path
                     if isinstance(parsed_result['body'], dict):
                         response._content = json.dumps(parsed_result['body'])
                     else:
-                        response._content = to_bytes(parsed_result['body'])
+                        body_bytes = to_bytes(parsed_result['body'])
+                        if parsed_result.get('isBase64Encoded', False):
+                            body_bytes = base64.b64decode(body_bytes)
+                        response._content = body_bytes
                 except Exception:
+                    LOG.warning("Couldn't set lambda response content")
```

Comment:
nitpick: Perhaps it would be useful to include some exception details in here as well:
```
                except Exception as e:
                    LOG.warning("Couldn't set lambda response content: %s" % e)
```